src/trackers/UNIT3D/DT.py

```
@@ -5,24 +5,17 @@
 
 
 class DT(UNIT3D):
-    """
-    DesiTorrents (DT) Tracker Class
-    """
-
+    tracker = "DT"
+    base_url = "https://torrent.desi"
+    banned_groups = ["DusIcTv", "PDHM", "Ranvijay", "BWT", "DDH", "Telly", "YTS", "RARBG", "BonsaiHD", "GalaxyRG", "-=!DrSTAR!=-"]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)